src/vite-env.d.ts

```diff
@@ -1 +1,11 @@
 /// <reference types="vite/client" />
+
+declare module "*.gql" {
+  const content: any;
+  export default content;
+}
```

Comment:
Is it possible to type these like this?

```suggestion
declare module "*.gql" {
  import { DocumentNode } from "graphql";
  const content: DocumentNode;
  export default content;
}
```